### ISSUE TITLE
Explore Page

### ISSUE THREAD

**Issue by Alonski** [Alonski/CoinPanion#7]
Have the ability to find Supportees by different criteria:
- Genre
- Keywords
- Most Supported

More?

**Comment by Josephrp** [comment]
One search bar, three classes, is plenty enough. Most supported... maybe : badges instead? see issue.

**Comment by Alonski** [comment]
If we implement badges then we can search by badges. #12

**Comment by rhlsthrm** [comment]
I put it in the mockup but I think the "Explore" page should be a drop down where you can explore by Top, Trending, New and then have a choice to browse by category. The categories can copy directly from Patreon.

**Comment by imbaniac** [comment]
If we want to make Explore Page with diff criteria's we'll have to add some kind of tags for supportées projects. Same for #9

**Comment by rhlsthrm** [comment]
> we'll have to add some kind of tags for supportées projects

Agreed, the user should be able to tag themselves with a category when they are creating their profiles.

**Comment by Josephrp** [comment]
Agreed. Solution is fine I think it's a good start and allows us to scale
down the road.
